Library/Formula/ant.rb

```diff
@@ -1,7 +1,7 @@
 class Ant < Formula
-  homepage "http://ant.apache.org/"
-  url "http://www.apache.org/dyn/closer.cgi?path=ant/binaries/apache-ant-1.9.4-bin.tar.gz"
-  sha1 "6c41481e8201f6b3f7e216146b95bb6de70208bb"
+  homepage "https://ant.apache.org/"
+  url "https://www.apache.org/dyn/closer.cgi?path=ant/binaries/apache-ant-1.9.4-bin.tar.bz2"
```

Comment:
There's no institutional preference. It's harder to verify the hash hasn't really changed, if nothing else.